packages/components/link/tests/link.test.tsx

```diff
@@ -31,3 +31,67 @@ describe("<Link />", () => {
     expect(link).toHaveTextContent("Welcome page")
   })
 })
+
+describe("<LinkOverlay />", () => {
+  test("renders correctly", () => {
+    render(
+      <LinkOverlay href="https://hirotomoyamada.github.io/yamada-ui/?path=/docs/documents-welcome--docs">
```

Comment:
@taroj1205 

Thanks PR.

I think it would be better to make the URL passed to href a constant to eliminate the redundant parts.

```diff
test("renders correctly", () => {
+  const url = "https://hirotomoyamada.github.io/yamada-ui/?path=/docs/documents-welcome--docs"
  render(
-    <LinkOverlay href="https://hirotomoyamada.github.io/yamada-ui/?path=/docs/documents-welcome--docs">
+    <LinkOverlay href={url}>
      Welcome page
    </LinkOverlay>,
  )
  const link = screen.getByRole("link")

  expect(link).toBeInTheDocument()
  expect(link).toHaveAttribute(
    "href",
-   "https://hirotomoyamada.github.io/yamada-ui/?path=/docs/documents-welcome--docs",
+    url,
  )
  expect(link).toHaveTextContent("Welcome page")
})
```